frog/imports/api/activities.js

```diff
@@ -39,50 +50,50 @@ export const addActivity = (activityType, data = {}, id, groupingKey) => {
   }
 };
 
-export const duplicateActivity = activity =>
+export const duplicateActivity = (activity: ActivityDbT) =>
   Activities.insert({
     ...activity,
     _id: uuid(),
     data: { ...activity.data, name: activity.data.name + ' (copy)' }
```

Comment:
I thought we had already separated it out, but I think we should move name to the "top-level", just like grouping key is (I think?)... data should be an opaque config that is only understandable by the activityType.